### DISCUSSION TITLE
Findings yaml import

### DISCUSSION THREAD

**Discussion by moscowchill** [pwndoc/pwndoc#150] [General]
Hello Guys,

I love pwndoc!
Can anyone help me with speeding up the configuration and point me to a collection of vulnerabilities so I do not have to manually import each and everyone of them?

Regards,
S

**Comment by azmuth94** [comment]
The serpico project has an entire folder of them that you can import. https://github.com/SerpicoProject/Serpico/tree/master/templates

**Comment by ogpadre** [comment]
Hello, did you find a solution to this problem? I am also very interested in this question.

**Comment by moscowchill** [comment]
No we ended up actually using our own findings and just adding more over 
time.

ogpadre schreef op 2022-06-17 13:07:
